downstream/modules/platform/proc-controller-importing-subscriptions.adoc

```diff
@@ -6,12 +6,12 @@ After you have obtained an authorized {PlatformNameShort} subscription, you must
 .Prerequisites
 
 * You have obtained a subscriptions manifest. 
-For more information, see xref:proc-controller-obtaining-subscriptions-manifest[Obtaining a subscriptions manifest].
+For more information, see link:https://access.redhat.com/documentation/en-us/red_hat_ansible_automation_platform/2.4/html-single/getting_started_with_automation_controller/index#controller-managing-subscriptions[Obtaining a subscriptions manifest].
```

Comment:
May be worth considering using the {BaseURL} attribute here and {PlatformVers} to future proof the links

Reply:
Good spot, updated.